inc/umock_c_internal.h

```diff
@@ -11,6 +11,7 @@ extern "C" {
 #include <stdlib.h>
 #endif
 
+#include <windows.h>
```

Comment:
a wild windows.h appears.

Reply:
Removed. I added it to test something.

---
In reply to: [266210477](https://github.com/Azure/umock-c/pull/55#discussion_r266210477) [](ancestors = 266210477)